t/cli/test_main.sh

```diff
@@ -211,6 +211,32 @@ fi
 
 echo "passed: resolve variables"
 
+# support reserved environment variable APISIX_DEPLOYMENT_ETCD_HOST
+
+echo '
+deployment:
+  role: traditional
+  role_traditional:
+    config_provider: etcd
+  etcd:
+    host:
+      - "http://127.0.0.1:2379"
+' > conf/config.yaml
+
+out=$(APISIX_DEPLOYMENT_ETCD_HOST='["http://127.0.0.1:2333"]' make init 2>&1 || true)
+if ! echo "$out" | grep "connection refused" > /dev/null; then
+    echo "failed: failed to configure etcd host with reserved environment variable"
+    exit 1
+fi
+
+out=$(APISIX_DEPLOYMENT_ETCD_HOST='["http://127.0.0.1:2333"]' make run 2>&1 || true)
```

Comment:
Can you configure a connectable address and check if it works?

Reply:
Done.